commons/python/business_solutions_api/dataiku_formula.py

```diff
@@ -9,18 +10,33 @@ def __init__(self):
         self._expressions = []
 
     @staticmethod
-    def _create_filter_by_column_vals(column: "str", vals: "list[str]"):
+    def _create_filter_by_column_vals(column: str, vals: Union["list[str]", dict]):
```

Comment:
I would create two sepparate functions - one `_create_filter_by_column_vals` and another `_create_filter_by_column_range` and choosing which one to use shouldn't be part of dataikuFormula class logic I think

Reply:
it is a possibility but I don't think there is a right or wrong way of doing it as both have their pros and cons